src/page.ts

```diff
@@ -262,6 +262,17 @@ export class Page extends EventTarget {
     );
   }
 
+  /**
+   * `page.setViewportSize()` will resize the page. A lot of websites don't expect phones to change size, so you should set the viewport size before navigating to the page.
+   */
+  async setViewportSize(size: { width: number; height: number }) {
+    await this.#celestial.Emulation.setDeviceMetricsOverride({
+      ...size,
+      deviceScaleFactor: 0,
```

Comment:
Why can't we set a device scale factor? I'm trying to migrate this Puppeteer based script that screenshots components at 4x resolution, and it doesn't seem easy with Astral:

https://github.com/jaydenseric/device-agnostic-ui-website/blob/8cc76b0a7733dcb4da4729b42e5d9a91405a361f/scripts/screenshots.mjs#L51-L55